Editor/AGS.Editor/AGSEditor.cs

```diff
@@ -1235,6 +1237,8 @@ public CompileMessages CompileGame(bool forceRebuild, bool createMiniExeForDebug
 
             Utilities.EnsureStandardSubFoldersExist();
 
+            if (RequiresRebuild) forceRebuild = true;

```

Comment:
I believe this does the same, alternative way of writing that's more concise. Not very important so feel free to ignore if you can't be bothered

```suggestion
            forceRebuild |= RequiresRebuild;
```